package.json

```diff
@@ -6,14 +6,14 @@
   "dependencies": {
```

Comment:
Please update version of package.json and run npm install

Reply:
Performed per review.